lib/rules/no-shadow.js

```diff
@@ -117,6 +118,29 @@ module.exports = {
             return def && def.name.range;
         }
 
+        /**
+         * Get declared line and column of a variable.
+         * @param {eslint-scope.Variable} variable The variable to get.
+         * @returns {Object|undefined} The declared line and column of the variable.
```

Comment:
Thankfully this can't ever be `undefined` because it always goes through either the `if` or `else` branch.

```suggestion
         * @returns {Object} The declared line and column of the variable.
```

Reply:
I removed `undefined`.